I2C.bme280.oled/simple_remote.py

```diff
@@ -0,0 +1,159 @@
+yanpiws_temp_1 = args.temp_id1
+yanpiws_ajax_url = 'http://' + str(yanpiws_ip) + '/ajax.php?content='
+last_seen_temp= 0.0
+
+# set up syslog logging
+my_logger = logging.getLogger('MyLogger')
+my_logger.setLevel(logging.DEBUG)
+handler = logging.handlers.SysLogHandler(address='/dev/log')
+my_logger.addHandler(handler)
+
+# BME280 sensor address (default address), Initialize I2C bus and calibration
+address = 0x76 # can also be 0x77 - check  i2cdetect -y 1 or i2cdetect -y 0
+bus = smbus2.SMBus(1) # can also be 0, depends on where you found device on per i2cdetect
+calibration_params = bme280.load_calibration_params(bus, address)
+
+def post_to_yanpiws():
+    data = bme280.sample(bus, address, calibration_params)
+    temperature_celsius = data.temperature
+    temperature_fahrenheit = round((temperature_celsius * 9 / 5) + 32, 2)
+    url = 'http://' + str(yanpiws_ip) + '/parse_and_save.php'
+    data = {
+        "model" : "BMP280",
+        "time" : datetime.now(tz=tz.tzlocal()).strftime("%Y-%m-%d %H:%M:%S"),
+        "id" : 71, # E bedroom, todo - not hardcode
+        "temperature_F" : temperature_fahrenheit,
+        "password" : "boxcar-spinning-problem-rockslide-scored"  # default pass, todo - not hardcode
+    }
+    global last_seen_temp
+    last_seen_temp = temperature_fahrenheit
+    requests.post(url, data=data)
+
+def get_string_from_url(url):
+    import urllib.request
+    raw_html = urllib.request.urlopen(url).read().decode('utf-8').rstrip()
+    return raw_html
+
+
+def get_remote_humid_and_temp(id):
+    forecast_url = yanpiws_ajax_url + 'humidity&id=' + str(id)
+    return json.loads(get_string_from_url(forecast_url))
+
+
+def get_remote_forecast():
+    forecast_url = yanpiws_ajax_url + 'forecast_full_json'
+    return json.loads(get_string_from_url(forecast_url))
+
+
+def get_remote_sun():
+    data = json.loads(get_string_from_url(yanpiws_ajax_url + "sunrise"))
+    data.update(json.loads(get_string_from_url(yanpiws_ajax_url + "sunset")))
+    return "☀" + str(data['sunrise']) + " ○" + str(data['sunset'])
+
+
+def show_info(wait):
+
+    # fetch the cooked up json -> strings
+    forecast = {}
+    humid_and_temp1 = {}
+    try:
+        no_error = True
+        humid_and_temp1 = get_remote_humid_and_temp(yanpiws_temp_1)
+        forecast = get_remote_forecast()
+        first_line = get_remote_sun()
+    except:
+        first_line = 'Error - check logs :('
+        no_error = False
+
+    second_line = ''
+    third_line = ''
+
+    if no_error and humid_and_temp1[0]['temp'] != 'NA':
+        second_line = str(int(float(humid_and_temp1[0]['temp']))) + '°' + humid_and_temp1[0]['label']
+
+    if no_error  and  forecast[0]:
+        third_line = str(int(float(forecast[0]['temperatureMax']))) + '°' + forecast[0]['icon']
+
+    full_path = os.path.dirname(os.path.abspath(__file__)) + "/"
+    font2 = ImageFont.truetype(full_path + "Lato-Heavy.ttf", 14)
+    font1 = ImageFont.truetype(full_path + "Lato-Heavy.ttf", 20)
```

Comment:
Consider [pathlib](https://docs.python.org/3/library/pathlib.html) instead of importing os. Like requests, it's a more pleasant abstraction.

All of the file handling code could be replaced with something like `font_file = pathlib.PurePath(__file__).parent() / "Lato-Heavy.ttf"`